activesupport/lib/active_support/railtie.rb

```diff
@@ -49,6 +49,12 @@ class Railtie < Rails::Railtie # :nodoc:
       Date.beginning_of_week_default = beginning_of_week_default
     end
 
+    initializer "active_support.require_master_key" do |app|
+      if app.config.respond_to?(:require_master_key) && app.config.require_master_key
+        app.credentials.key
```

Comment:
It's a little hidden that `key` will ensure the key is there and raise if it isn't. Though adding a method that simply aliased `key` seemed off too.

Actually instead of raising we should catch the exception, print its message and `exit 1` — spares us a misleading backtrace too. I'll do that now.